tools/c-parser/standalone-parser/Makefile

```diff
@@ -53,8 +53,11 @@ include $(STP_PFX)/../Makefile
 STP_CLEAN_TARGETS := $(STPARSERS) $(TOKENIZERS) $(STP_PFX)/c-parser.o $(STP_PFX)/table.ML
 
 $(STP_PFX)/table.ML: $(ISABELLE_HOME)/src/Pure/General/table.ML $(STP_PFX)/Makefile
-	sed -e '/\(\* cache \*\)/,/final.declarations/d' < $< | \
-	sed -e "s/^  val unsynchronized_cache:.*a/  (* removed unsynchronized_cache *)/" > $@
+	sed -e '/\(\* cache \*\)/,/final declarations of this structure/d' < $< | \
+  sed -e "s/^  type 'a cache_ops.*int./  (* removed type 'a cache_ops *)/" | \
+  sed -e "s/^  val unsynchronized_cache:.*cache_ops/  (* removed unsynchronized_cache *)/" | \
+  sed -e "s/^  val apply_unsynchronized_cache.*a/  (* removed apply_unsynchronized_cache *)/" > $@
+
```

Comment:
I'm trying to understand this and can't... where is the first sed getting its inputs?
Also, do you want the extra newline after?

Reply:
The input is `< $< ` (Makefile `$< `). This part is mostly black sed magic with Makefile fairy dust sprinkled on. It removes some bits from the Isabelle implementation of `table.ML` that would draw in more dependencies. The really gross one is the `sed` in the `mk_release` which operates on this Makefile and removes the `sed` here..

This time I did actually want the additional line, the next set of targets are a different kind, so I wanted a new section.